src/search/search.component.ts

```diff
@@ -176,7 +182,9 @@ export class Search implements ControlValueAccessor {
 	 * @param search The input text.
 	 */
 	onSearch(search: string) {
-		this.value = search;
+		if (!this.isComposing) { // check for IME use
+			this.value = search;
+		}
 		this.doValueChange();
```

Comment:
Do we need to do `this.doValueChange()` if it's composing? If not, let's guard it:
```suggestion
		if (this.isComposing) { // check for IME use
			return;
		}
		this.value = search;
		this.doValueChange();
```

If we do, it's fine.

Reply:
Double checked and seems like guarding it still works! 👍